packages/flutter/lib/src/widgets/async.dart

```diff
@@ -193,46 +201,61 @@ enum ConnectionState {
 ///    with a [Future].
 @immutable
 class AsyncSnapshot<T> {
-  /// Creates an [AsyncSnapshot] with the specified [connectionState],
-  /// and optionally either [data] or [error] (but not both).
-  const AsyncSnapshot._(this.connectionState, this.data, this.error)
+  /// Creates an [AsyncSnapshot] with the specified [connectionState] and
+  /// [hasData], and optionally either [data] or [error] (but not both).
```

Comment:
should it be `[_data]` as it refers to parameter here?

Reply:
No, I'm linking to the property intentionally.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements